core/src/test/resources/DeploymentSpecifics.properties

```diff
@@ -1,4 +1,4 @@
 instance.name=test
-log.dir=${user.dir}\\target
+log.dir=${user.dir}\\target\\test-logs
```

Comment:
Bedoel je niet build.dir? user.dir is toch niet de project directory?

Reply:
user.dir is the directory where java was run from. When running tests, it's the root directory of either the IAF module, or the entire IAF framework.